tests/Sharpliner.Tests/AzureDevOps/TemplateTests.cs

```diff
@@ -248,6 +447,237 @@ public void Stage_Template_Definition_Serialization_Test()
             """);
     }
 
+    private class Stage_Typed_Template_Definition(StageTypedParameters? parameters = null) : StageTemplateDefinition<StageTypedParameters>(parameters)
+    {
+      public override string TargetFile => "stage-template.yml";
+
+        public override ConditionedList<Stage> Definition => 
+        [
+          ..new Stage_Template_Definition().Definition,
```

Comment:
Quick question about this one: should we add an implicit cast so that just `new()` is enough?

Reply:
This is just me being lazy on the tests and reusing the definition